workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/AccountsTicketsRestController.java

```diff
@@ -46,34 +50,39 @@ public class AccountsTicketsRestController extends BaseRestController {
 	)
 	public ResponseEntity<String> getZendeskTickets(
 			@AuthenticationPrincipal Jwt jwt,
-			@PathVariable("externalReferenceCode") String externalReferenceCode)
+			@PathVariable("externalReferenceCode") String externalReferenceCode,
+			@RequestParam(defaultValue = "") long[] associatedTicketIds,
+			@RequestParam(defaultValue = "false") boolean includeAllOpenTickets)
```

Comment:
I dont think we need "all" in this name since it's somewhat already implied and there's no other openTicket option